doc/man/scons.xml

```diff
@@ -924,6 +924,21 @@ of a given derived file:</para>
 
 <screen>
 $ <userinput>scons --debug=includes foo.o</userinput>
+</screen>
+
+  </listitem>
+  </varlistentry>
+
+<varlistentry>
+  <term><emphasis role="bold">json</emphasis></term>
+  <listitem>
+<para>Write info to a JSON file for any of the following debug options if they are enabled: <emphasis>memory</emphasis>,
+    <emphasis>count</emphasis>, <emphasis>time</emphasis>, <emphasis>action-timestamps</emphasis> </para>
+    <para>The default output file is in the top level directory named <literal>scons_stats.json</literal></para>
+    <para>The file name/path can be modified by using <literal>DebugOptions(json='path/to/file.json')</literal></para>
```

Comment:
use link here. Suggest: "The name/path of the debug output file can be set using the `&f-link-DebugOptions;` function.

Reply:
Added link and then prefixed example with "for example"